src/main/scala/org/zalando/kanadi/api/Subscriptions.scala

```diff
@@ -1061,12 +1060,14 @@ case class Subscriptions(baseUri: URI, authTokenProvider: Option[AuthTokenProvid
   }
 
   private final val killSwitches =
-    new ConcurrentHashMap[(SubscriptionId, StreamId), UniqueKillSwitch]().asScala
+    new ConcurrentHashMap[(SubscriptionId, StreamId), UniqueKillSwitch]()
 
   def addStreamToKillSwitch(subscriptionId: SubscriptionId,
                             streamId: StreamId,
-                            uniqueKillSwitch: UniqueKillSwitch): Unit =
-    killSwitches((subscriptionId, streamId)) = uniqueKillSwitch
+                            uniqueKillSwitch: UniqueKillSwitch): Unit = {
+    killSwitches.put((subscriptionId, streamId), uniqueKillSwitch)
+    ()
```

Comment:
Needed because `ConcurrentHashMap.put` doesn't return `unit` so by putting a manual `()` we are telling the scala compiler we in fact don't care about what `ConcurrentHashMap.put` returns